### ISSUE TITLE
BorderlessWindow:minimize(true) can not work on macOS

### ISSUE THREAD

**Issue by gqf2008** [Vadoola/Tomotroid#74]
slint issues [#4685](https://github.com/slint-ui/slint/issues/4685)
winit issues Fixed [#1411](https://github.com/rust-windowing/winit/pull/1411)

**Comment by gqf2008** [comment]
it work
```
let mut backend = i_slint_backend_winit::Backend::new().unwrap();
    if cfg!(target_os = "macos") {
        use i_slint_backend_winit::winit::platform::macos::WindowBuilderExtMacOS;
        backend.window_builder_hook = Some(Box::new(|builder| {
            builder.with_decorations(false)
        }));
    }

    slint::platform::set_platform(Box::new(backend)).unwrap();
```

**Comment by Vadoola** [comment]
Thank you, I really appreciate it.

**Comment by Vadoola** [comment]
This was closed in Commit 2662b96
